public/pages/SplitIndex/container/SplitIndex/__snapshots__/SplitIndex.test.tsx.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<SplitIndex /> spec renders the component 1`] = `
+HTMLCollection [
+  <div>
+    <div
+      style="padding: 0px 50px;"
+    >
+      <h1
+        class="euiTitle euiTitle--large"
+      >
+        Split index
+      </h1>
+      <div
+        class="euiSpacer euiSpacer--l"
+      />
+    </div>
+  </div>,
+]
+`;
```

Comment:
why the snapshot only contains a title, should there have some content?

Reply:
Good catch, I didn't pay attention to snapshot file, as long as it didn't report an error, will take a look at it later.